airflow/operators/presto_to_mysql.py

```diff
@@ -28,19 +28,19 @@ class PrestoToMySqlTransfer(BaseOperator):
     into memory before being pushed to MySQL, so this operator should
     be used for smallish amount of data.
 
-    :param sql: SQL query to execute against Presto
+    :param sql: SQL query to execute against Presto. (templated)
     :type sql: str
     :param mysql_table: target MySQL table, use dot notation to target a
-        specific database
+        specific database. (templated)
     :type mysql_table: str
     :param mysql_conn_id: source mysql connection
     :type mysql_conn_id: str
     :param presto_conn_id: source presto connection
     :type presto_conn_id: str
     :param mysql_preoperator: sql statement to run against mysql prior to
-        import, typically use to truncate of delete in place of the data
-        coming in, allowing the task to be idempotent (running the task
-        twice won't double load data)
+        import (templated), typically use to truncate of delete in place
```

Comment:
same here. Please add `(templated)` at the end of the line instead of before comma `,`